vaxrank/vaxrank_results.py

```diff
@@ -105,29 +95,32 @@ def variant_properties(self, gene_pathway_check=None):
         is a strong MHC binder, etc.
         """
         variant_properties_list = []
-        for variant in self.variants:
-            gene_name = ''
-            if variant.gene_names:
-                gene_name = variant.effects().top_priority_effect().gene_name
+        for isovar_result in self.isovar_results:
+            variant = isovar_result.variant
+            if isovar_result.top_protein_sequence:
+                gene_name = isovar_result.top_protein_sequence.gene_name
+            else:
+                gene_name = isovar_result.overlapping_gene_names(only_coding=False)
+
             variant_dict = OrderedDict((
                 ('contig', variant.contig),
                 ('start', variant.start),
                 ('ref', variant.ref),
                 ('alt', variant.alt),
-                ('is_coding_nonsynonymous', False),
-                ('rna_support', False),
-                ('mhc_binder', False),
+                ('is_coding_nonsynonymous',
+                    isovar_result.predicted_effect_modifies_protein_sequence),
+                ('rna_support',
```

Comment:
wooo! Just checking: `rna_support` will now be true if there's RNA evidence even if there are no MHC binders, yes?

Reply:
Correct